validator/src/server/cli.rs

```diff
@@ -221,15 +221,15 @@ pub fn parse_args<'a>() -> ArgMatches<'a> {
     app.get_matches()
 }
 
-#[allow(clippy::needless_pass_by_value)]
+#[allow(needless_pass_by_value)]
```

Comment:
This change in the commit is unnecessary. Solution: Squash the other commit that is reverting this part into the same commit.